pallets/migration/src/lib.rs

```diff
@@ -76,11 +76,22 @@ pub mod pallet {
 
 		/// WeightInfo
 		type WeightInfo: WeightInfo;
+
+		type Filter: Contains<<Self as frame_system::Config>::Call>;
 	}
 
 	#[pallet::hooks]
 	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {}
 
+	#[pallet::type_value]
```

Comment:
For my education, what does this mean?

Reply:
Nor sure. Would bet it automatically implements some trait so that we can use this function when our storage code does not exist yet and fails to decode in sp_io.

Reply:
TIL: https://crates.parity.io/frame_support/attr.pallet.html#type-value-pallettype_value-optional